src/components/grid/_index.scss

```diff
@@ -0,0 +1,9 @@
+@use "../../scss";
+
+.c-grid {
+	--grid-template-columns: var(--c-grid-template-columns, repeat(auto-fit, minmax(15em, 1fr)));
```

Comment:
My only concern here is the min-width.  I grabbed that value from: https://codepen.io/web-dot-dev/pen/QWMMVPm so if that sort of minimum width value (15em ~= 240px) makes sense, then we can stick with it.
![image](https://user-images.githubusercontent.com/2287238/163275187-8bc065a8-df52-4cb6-ac47-d6985608c069.png)

15em breaks to single column on the iPhone 12:
![image](https://user-images.githubusercontent.com/2287238/163275566-051eb1b7-6f7e-42c8-9fb2-e9d399c8b3d1.png)

10em breaks to two:
![image](https://user-images.githubusercontent.com/2287238/163275640-89e10984-7128-45fb-8d35-08188be2177f.png)

I don't know that I like 2 columns on such a small device, but the mobile design is 2:
![image](https://user-images.githubusercontent.com/2287238/163275850-704e4384-54f7-4312-94ea-b949bf9d280f.png)

Reply:
depending on gap, 32% actually defaults to 2 column on mobile and broadens to 3 columns on everything else.

Reply:
I played around with the min value and came to the conclusion `15em` being ~240px as the min width of a grid children item seemed to be a sane default